packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md

```diff
@@ -176,9 +176,9 @@ The following table provides a list of all allowed HTML tags and elements in cus
 
 ## Add translations
 
-**Note:** After you customize a template in one language, you need to customize other languages if you want to include them. If you reset all customized translations, then Okta uses the 27 default translations.
+Okta uses the default language when the end user’s locale doesn't match any email customizations. You can edit the templates through the pencil icon. If you restore an email to default settings, you delete any customizations made to the email whether or not the custom email is your default.
```

Comment:
```suggestion
Okta uses the default language when the end user’s locale doesn't match any email customizations. You can edit the templates through the pencil icon. If you restore an email to default settings, you delete any customizations made to the email whether or not the customization is for the default language.
```

Reply:
Not sure if that works better, you decide. :)

Reply:
sorry I missed this. adding it now :)